assets/scripts_ssh_config

```diff
@@ -1,3 +1,4 @@
 StrictHostKeyChecking no
 UserKnownHostsFile /dev/null
+PasswordAuthentication no
```

Comment:
There may be a more isolated way to do this, but the test hangs in WaitForSSH at a password prompt without it.